sites/feedback/ajax/page-feedback-contact-en.html

```diff
@@ -0,0 +1,69 @@
+		{ "selector": "[data-feedback-url]", "attr": "data-feedback-url", "path": "contact/url" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0">
+			<h3 class="wb-inv">Give feedback about this page</h3>
+			<form action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback wb-disable-allow" data-wb-postback='{"success":".gc-pft-thnk"}'>
+				<div class="wb-disable-allow" data-wb-json='{
+					"url": "#[feedback]",
+					"mapping": [
+						{ "selector": "input", "attr": "name", "value": "/@id" },
+						{ "selector": "input", "attr": "value", "value": "/@value" }
+					]
+				}'>
+					<template>
+						<input type="hidden" name="" value="" />
+					</template>
+				</div>
+				<fieldset class="gc-pft-btns chkbxrdio-grp row row-no-gutters d-sm-flex flex-sm-wrap align-items-sm-center">
+					<legend class="col-xs-12 col-sm-7 nojs-col-sm-12 col-md-9 col-lg-8 text-center text-sm-left nojs-text-left mrgn-tp-sm pr-sm-3 nojs-pr-sm-0"><span class="field-name">Did you find what you were looking for?</span></legend>
+					<div class="col-xs-12 nojs-show">
+						<button name="helpful" value="Yes-Oui" class="btn btn-primary" aria-describedby="why-note">Yes</button>
+					</div>
+					<div class="col-xs-12 col-sm-5 col-md-3 col-lg-4 text-center text-sm-right nojs-hide">
+						<button name="helpful" value="Yes-Oui" class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm">Yes</button>
+						<button class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm mrgn-lft-sm" data-wb-doaction='[
+							{"action":"removeClass","source":".gc-pft-no","class":"nojs-show"},
+							{"action":"addClass","source":".gc-pft-btns","class":"hide"}
+						]'>No</button>
+					</div>
+				</fieldset>
+				<div class="gc-pft-no nojs-show">
+					<p id="why-note" class="nojs-show mrgn-tp-lg mrgn-bttm-md"><b>If not, tell us why below:</b></p>
+					<p class="nojs-hide wb-inv" aria-live="polite"><b>Tell us why below:</b></p>
+					<details>
+						<summary>Need urgent help with a problem? Contact us</summary>
+						<p class="mrgn-bttm-0 mrgn-tp-md">
+							<a href="#" class="wb-disable-allow" data-wb-json='[
+							{ "url": "#[feedback]/contact/url", "type": "attr", "attr": "href" },
+							{ "url": "#[feedback]/contact/link", "type": "replace" }
+							]'></a>
+						</p>
+					</details>
+					<div class="form-group">
+						<label for="problem" class="mrgn-bttm-0"><span class="field-name">Please provide more details</span></label>
+						<p id="problem-note" class="mrgn-bttm-sm"><small><b>You will not receive a reply. Don't include personal information (telephone, email, SIN, financial, medical, or work details).</b></small></p>
+						<p><small>Maximum 300 characters</small></p>
+						<textarea id="problem" aria-describedby="problem problem-note" name="details" class="form-control full-width" maxlength="300"></textarea>
```

Comment:
The extended description should not refer to itself : aria-describedby="**problem** problem-note"

Reply:
@duboisp From what I understand, the intention here was to announce the label and then the note. Since the label and the textarea are connected, it announces the label when referencing the textarea in the "aria-describedby". That means that on focus of the textarea, the AT will announce: "Please provide more details. You will not receive a reply. Don't include personal information (telephone, email, SIN, financial, medical, or work details)."

I could add an ID to the label instead, if you prefer.

Reply:
As discussed, removed the self-reference. I also added the "Maximum 300 characters" instruction in the aria-describedby.